x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/log_rate_analysis_results_table_groups.tsx

```diff
@@ -423,6 +423,26 @@ export const LogRateAnalysisResultsGroupsTable: FC<LogRateAnalysisResultsTablePr
     };
   }, [pageIndex, pageSize, sortField, sortDirection, groupTableItems]);
 
+  // If no row is hovered or pinned, fall back to set the first row
+  // into a hovered state to make the main document count chart
+  // show a comparison view by default.
+  useEffect(() => {
```

Comment:
If you change the field filter, the first row isn't being selected when the data reloads.

Reply:
Fixed in [33a06be](https://github.com/elastic/kibana/pull/162941/commits/33a06be10ebc18345720d8205d56251d0de5f676). I forgot to apply the fix I create for the regular table to the grouped one.